pkg/module/disk_ref.go

```diff
@@ -31,14 +34,29 @@ func newDiskRef(fs afero.Fs, root, version string) *diskRef {
 //
 // You should always call this function after you fetch a module into a DiskRef
 func (d *diskRef) Clear() error {
+
+	// This is required because vgo ensures dependencies are read-only
+	// See https://github.com/golang/go/issues/24111 and
+	// https://go-review.googlesource.com/c/vgo/+/96978
+	walkFn := func(path string, info os.FileInfo, err error) error {
```

Comment:
in unix you can pass an `-R` flag and it will chmod all the sub files and directories. Is there an equivalent in Go? just wondering so we don't have to walk the whole dir with potentially many files.

Reply:
I tried looking but was unsuccessful - do you know of something?